x/consensus/keeper/consensus/batch_test.go

```diff
@@ -43,7 +45,7 @@ func TestBatching(t *testing.T) {
 			QueueTypeName: "simple-message",
 			Sg:            sg,
 			Ider:          keeperutil.NewIDGenerator(sg, nil),
-			Cdc:           types.ModuleCdc,
+			Cdc:           nil,
```

Comment:
I think this will create problems. The unpacker is used by different keeper methods.